cmd/create/runner.go

```diff
@@ -206,6 +206,16 @@ func (r *runner) run(ctx context.Context, _ *cobra.Command, _ []string) error {
 	}
 	r.logger.LogCtx(ctx, "message", "created release CR")
 
+	// Write release ID to filesystem
+	{
+		releaseIDPath := filepath.Join(r.flag.Output, "release-id")
```

Comment:
Not blocking as this is not new: Why did we decide to use the filesystem for these outputs? Isn't there a way in tekton to define outputs (especially simple strings) more nicely and pass them between tasks?

Reply:
Yep, there are [results](https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md#emitting-results) but you also have to write files for them.